app/lib/pdf/report.rb

```diff
@@ -53,7 +53,7 @@ def generate
       metrics
       move_down 3
 
-      @report = @pdf.render_file(Rails.root.join('tmp', "#{@account.domain} #{@service.name}.pdf"))
+      @report = @pdf.render_file(Rails.root.join('tmp', "#{@account.domain} #{@service.id}.pdf"))
```

Comment:
We could also do something like 
```suggestion
      @report = @pdf.render_file(Rails.root.join('tmp', "pdf-report-#{@account.domain}-#{@service.id}.pdf"))
```

This way no spaces ...

Reply:
why we don't use the `pdf_file_name` method?
https://github.com/3scale/porta/blob/46f6bdb1963498e8a4e4b49a15ec110f2e01e048/app/lib/pdf/report.rb#L88-L90